src/App.js

```diff
@@ -96,8 +96,10 @@ const Guess = ({ guess, matches, word }) => {
         const animation = [];
         if (word) {
           animation.push({ type: "fadeIn", delay: 500 * index });
-          if (word === guess)
+          if (word === guess) {
```

Comment:
Can merge these two lines to be:
```
if (word === guess && !giveup)
```

Reply:
Yup, should have done that to begin with